Makefile

```diff
@@ -11,6 +11,12 @@ BUILD_DIR = build
 BUILD_TOOLS_DIR = $(BUILD_DIR)/tools
 GO_ENV_VARS = GO111MODULE=on GOOS=windows
 CSI_PROXY_API_GEN = $(BUILD_DIR)/csi-proxy-api-gen
+VERSION=$(shell git describe)
+REV=$(shell git describe --long --tags --match='v*' --dirty 2>/dev/null || git rev-list -n1 HEAD)
+
+$(info PULL_BASE_REF is $(PULL_BASE_REF))
```

Comment:
it doesn't look like're using these values in the bucket path. Is that right?

Reply:
actually I am still testing and verifying this part, so put WIP in this PR. sorry for the confusion.